### DISCUSSION TITLE
The strange completion window which stays there forever.

### DISCUSSION THREAD

**Discussion by hongyi-zhao** [company-mode/company-mode#1378] [Q&A]
See my following screenshot:

![image](https://user-images.githubusercontent.com/11155854/229356667-e6621f74-b800-4b5f-9bcf-083ef49366fe.png)

As you can see, I got a strange completion window which stays there forever.

Any tips for fixing this problem?

Regards,
Zhao

**Comment by dgutov** [comment]
Is that with the `company-posframe` frontend?

**Reply by hongyi-zhao** [reply]
Yep. I've the following setting in `~/.emacs.d/init.el`:
```elisp
(use-package company-posframe
  :hook (company-mode . company-posframe-mode)
  ;;https://github.com/tumashu/company-posframe/blob/c7a820a35ff132aaec53c81e05afc829de39eb68/company-posframe.el#L117
  :custom (company-posframe-quickhelp-delay nil))
```

**Reply by dgutov** [reply]
Try filing a bug report there. Ideally, with a reproduction scenario.

**Reply by hongyi-zhao** [reply]
I eventually disabled the `posframe` package, which is just for decorative purposes, IMO.
